### PR TITLE
angle_to_sexigesimal bugfix

### PR DESCRIPTION
**Pull request by stephenportillo**
Fixed degree-to-hour conversion ratio and made HMS printout zero padded

### CODE REVIEW

File: Portillo/template/buggy_function.py [hunk 1]
```diff
@@ -20,13 +20,13 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     if math.floor(decimals) != decimals:
         raise OSError('decimals should be an integer!')
 
-    hours_num = angle_in_degrees*24/180
+    hours_num = angle_in_degrees*24/360
```
Comment (tahumada) [inline Portillo/template/buggy_function.py:23]: I'll take a look at it

Comment (tahumada) [reply Portillo/template/buggy_function.py:23]: It works fine!

**Review by tahumada — CHANGES_REQUESTED** [review]
Just trying!

**Comment by tahumada** [comment]
Hey, trying again

**Comment by stephenportillo** [comment]
Hey, thanks for reviewing my code!

**Review by tahumada — APPROVED** [review]

**Comment by tahumada** [comment]
@astrofrog  @stephenportillo's code is ready to be merged!

**Comment by bsipocz** [comment]
@stephenportillo @tahumada - Thanks for opening the pull request and for the review(s)! Everything looks good, so we'll go ahead and merge this. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. 

@tahumada - your comment seems generic and not about the specific code changes in the "Files changed" tab. For these cases it's possible to leave the generic comments when you set the review status (Approve/Require Changes/etc.), or in the conversation tab, here rather than inline.

**Pull request merged**